### DISCUSSION TITLE
表单设计器是否支持配置数据源关联

### DISCUSSION THREAD

**Discussion by kkodeok** [alibaba/formily#2123] [Q&A 2.x] [answered]
假设有两个select，对于第一个select用effect异步拿到了DataSource ，但是想共享给第二个select，但是在设置依赖字段时发现字段属性中并没有DataSource这个属性，请问是本身就不支持这么操作吗？

**Comment by janryWang** [comment, marked as answer]
响应器配置

**Reply by kkodeok** [reply]
不好意思  可能是我描述的有问题， 确实是说的影响器配置的场景但是在这里没有找到DataSource属性
![image](https://user-images.githubusercontent.com/39008983/132489993-a0b1f7b8-858e-44dd-9272-acf853e44a37.png)
